src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -77,7 +71,7 @@ GHPullRequest wrapUp(GitHub root) {
 
     @Override
     protected String getApiRoute() {
-        return "/repos/"+owner.getOwnerName()+"/"+owner.getName()+"/pulls/"+number;
+        return "/repos/" + owner.getOwnerName() + "/" + owner.getName() + "/pulls/" + number;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)